global/scripts/MyelinMap_BC.sh

```diff
@@ -11,21 +11,23 @@ fi
 
 source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
 source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
+source "$HCPPIPEDIR/global/scripts/tempfiles.shlib"
 
 defaultSigma=$(echo "sqrt(200)" | bc -l)
 
 #description to use in usage - syntax of parameters is now explained automatically
-opts_SetScriptDescription "implements CIFTI-based Myelin bias fields from native mesh to low resolution then back to native mesh"
+opts_SetScriptDescription "implements CIFTI-based Myelin bias fields from native mesh to 32k resolution then back to native mesh; by default changing the group reference's mean with individual map's mean"
```

Comment:
I wouldn't mention 32k or low resolution here, that is an implementation detail, and the help info already lists the defaults.  I would have it just say "corrects an individual native mesh myelin map for bias fields based on a group average myelin map".